packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,13 +3,25 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.78.2
```

Comment:
nit: I've seen people ask for two new lines above the version number 🙈

Reply:
Fixed

Reply:
Fixed